src/modules/highlight/TextHighlighter.ts

```diff
@@ -990,43 +990,41 @@ export default class TextHighlighter {
         }
       }
 
-      if (oc(this.config).selectionMenuItems([])) {
-        oc(this.config)
-          .selectionMenuItems([])
-          .forEach((menuItem) => {
-            var itemElement = document.getElementById(menuItem.id);
-            var self = this;
-
-            function itemEvent() {
-              itemElement.removeEventListener("click", itemEvent);
-
-              function getCssSelector(element: Element): string {
-                const options = {
-                  className: (str: string) => {
-                    return _blacklistIdClassForCssSelectors.indexOf(str) < 0;
-                  },
-                  idName: (str: string) => {
-                    return _blacklistIdClassForCssSelectors.indexOf(str) < 0;
-                  },
-                };
-                return uniqueCssSelector(
-                  element,
-                  self.delegate.iframe.contentDocument,
-                  options
-                );
-              }
-
-              const selectionInfo = getCurrentSelectionInfo(
-                self.delegate.iframe.contentWindow,
-                getCssSelector
+      if (this.config?.selectionMenuItems ?? []) {
```

Comment:
maybe this if statement is obsolete, it was before as well, since the foreach would only run if the array is not empty

Reply:
Hmm yea maybe, but I think we should leave it for now and make that change in a follow up if we want to

Reply:
👍